CONTRIBUTING.md

```diff
@@ -2,24 +2,38 @@ Thanks for contributing!
 
 ## Development
 
-Run `npm run examples` to run a webpack dev server with Radium examples.
+### Installing dependencies
 
-## Lint
+```bash
+npm install
+```
 
-Run `npm run lint` before committing to lint files.
+You will find all building blocks that make up Radium in the [`modules`](modules) folder.
 
-## Flow
+### Testing
 
-1. [Install flow](http://flowtype.org/docs/getting-started.html#installing-flow)
-2. Run `flow` to check for missing annotations and static type check failures
+You will find tests for each module inside [`modules/__tests__`](modules/__tests__). Whenever making any changes, ensure that all existing tests pass by running `karma start`.
```

Comment:
`npm test-dev` instead of `karma start`? Since `npm install` will not install global karma.